webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateTimePicker.js

```diff
@@ -93,7 +93,7 @@ class DateTimePicker extends React.Component {
           <Popover
             position={placement}
             className="container"
-            style={{ minWidth: '50em', width: '33%'}}
+            style={{ minWidth: '50em', width: '33%' }}
```

Comment:
Something we can already merge, right?

Reply:
https://github.com/theforeman/foreman/pull/9901